src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestUpdater.cs

```diff
@@ -836,6 +840,49 @@ private async Task ClearAllStateAsync()
         await _pullRequestUpdateReminders.UnsetReminderAsync();
     }
 
+    /// <summary>
+    ///     Given a set of updates, replace the `from` version of every dependency update with the corresponding version
+    ///     from the target branch 
+    /// </summary>
+    /// <param name="darcRemote">Darc client used to fetch target branch dependencies.</param>
+    /// <param name="targetRepository">Target repository to fetch the dependencies from.</param>
+    /// <param name="targetBranch">Target branch to fetch the dependencies from.</param>
+    /// <param name="targetRepositoryUpdates">Incoming updates to the repository</param>
+    /// <returns>
+    ///     Subscription update and the corresponding list of altered dependencies
+    /// </returns>
+    /// <remarks>
+    ///     This method is intended for use in situations where we want to keep the information about the original dependency
+    ///     version, such as when updating PR descriptions.
+    /// </remarks>
+    private static async Task<List<(SubscriptionUpdateWorkItem update, List<DependencyUpdate> deps)>> ReplaceFromDependency(
+        IRemote darcRemote,
+        string targetRepository,
+        string targetBranch,
+        TargetRepoDependencyUpdate targetRepositoryUpdates
+        )
```

Comment:
```suggestion
        TargetRepoDependencyUpdate targetRepositoryUpdates)
```

There's an auto-refactoring for wrapping/indenting arguments that I use often that makes it this way btw